report.c

```diff
@@ -472,7 +472,7 @@ void csv_close(time_t now)
 
       /* 1000.0 is a temporay hack for stats usec to ms, impacted net_loss. */
       if( strchr( data_fields[j].format, 'f' ) ) {
-	printf( ",%.2f", (float) (data_fields[j].net_xxx(at) / 1000.0));
```

Comment:
Apparently I wasnt clear. 
`someint/1000.0
`promotes someint to a double before doing the calculation. The result is a double. Why would you cast that to float or double before passing it as an argument to printf?
you could make the "upgrade" of the integer explicit by saying:
`((double)someint) / 1000.0
`and in the case at hand because the someint is complex, you don't want the first part of that expression be casted but the whole thing. So it would become
`((double) (data_fi.....(at))) / 1000.0`